manifests/init.pp

```diff
@@ -49,6 +50,11 @@
   $service_ensure       = $::dns::params::service_ensure,
   $service_enable       = $::dns::params::service_enable,
   $additional_options   = $::dns::params::additional_options,
+  $transfert_source     = $::dns::params::transfert_source,
```

Comment:
`transfert_source` or `transfer_source`? The definition in the params class is transfer_source, but I think the directive is without the 't'.